projects/ama-sdk/src/lib/api-implementations/acm-api/dmn-api.ts

```diff
@@ -0,0 +1,70 @@
+ /*!
+ * @license
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Observable, from } from 'rxjs';
+import { Model } from '../../api/types';
+import { ModelApiInterface } from '../../api/generalmodel-api.interface';
+import { RequestApiHelper, RequestApiHelperOptions } from './request-api.helper';
+ import { ModelApiVariation } from '../acm-api/model-api';
+
+export class DmnApi<T extends Model, S> implements ModelApiInterface<T, S> {
+
+    constructor(protected modelVariation: ModelApiVariation<T, S>, protected requestApiHelper: RequestApiHelper) {}
+
+    public validate(modelId: string, content: S, requestOptions: any): Observable<any> {
+        return this.validateDmn(modelId, content, requestOptions);
+    }
+
+    private validateDmn(tableName: string, variables, dmnTable: any) {
+        const requestOptions: RequestApiHelperOptions = {
+            bodyParam: {
+                'name' : tableName,
+                'input' : variables,
+                'xml': this.modelVariation.serialize(dmnTable)
+            }
+        };
+        return from(this.requestApiHelper.post(`dmn-service/v1/dmn/execute`, requestOptions));
+    }
+
+    create(model: Partial<T>, containerId?: string): Observable<T> {
```

Comment:
Besides the validation, every method here seems like just dummy methods. I think it should be part of the DecisionTableApi, where we override the validation, the same way as we did for the ProcessApi.